### PR TITLE
#402 added information about missing or extra new lines

### PR DESCRIPTION
**Pull request by krzyk**
#402 
* recreated problem in invoker test (removed wrong test, and replaced it with correct one)
* fixed problem with missing new lines, by forcing split to output full list of tokens (by using `-1`)

### CODE REVIEW

**Comment by caarlos0** [comment]
@krzyk NICE!

**Comment by davvd** [comment]
@krzyk Many thanks for the PR, let me find a reviewer for it

**Comment by davvd** [comment]
@pinaf this pull request is for you,please review

File: qulice-maven-plugin/src/it/xml-violations/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2014-2015, Xockets, Inc.
```
Comment (pinaf) [inline qulice-maven-plugin/src/it/xml-violations/LICENSE.txt]: @krzyk this license file seems wrong.

File: qulice-maven-plugin/src/it/xml-violations/src/site.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/DECORATION/1.3.0 http://maven.apache.org/xsd/decoration-1.3.0.xsd"
+         name="foo-transport">
+    <skin>
+        <groupId>com.jcabi</groupId>
+        <artifactId>jcabi-maven-skin</artifactId>
+        <version>1.5.1</version>
+    </skin>
+    <bannerLeft>
+        <name>xockets.com</name>
```
Comment (pinaf) [inline qulice-maven-plugin/src/it/xml-violations/src/site.xml]: @krzyk same here. change this reference.

**Comment by pinaf** [comment]
@krzyk 2 minor comments

**Comment by krzyk** [comment]
@pinaf fixed

**Comment by pinaf** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by davvd** [comment]
@pinaf Thanks for your contribution, **17 mins** was added to your account, payment ID is `AP-67Y82115LR299804A`, 49 hours and 32 mins spent total. review comments (c=2) added as a bonus. added +17 to your rating, now it is equal to [+4275](http://www.netbout.com/b/35128?open=rating)

**Comment by davvd** [comment]
@rultor deploy

**Pull request merged**